Security/Authorization/Interception/MethodSecurityInterceptor.php

```diff
@@ -41,20 +39,41 @@
  */
 class MethodSecurityInterceptor implements MethodInterceptorInterface
 {
+    /** @var bool */
     private $alwaysAuthenticate;
-    private $securityContext;
+
+    /** @var TokenStorageInterface */
+    private $tokenStorage;
+
+    /** @var MetadataFactoryInterface */
     private $metadataFactory;
+
+    /** @var AuthenticationManagerInterface */
     private $authenticationManager;
+
+    /** @var AccessDecisionManagerInterface */
     private $accessDecisionManager;
+
+    /** @var AfterInvocationManagerInterface */
     private $afterInvocationManager;
+
+    /** @var RunAsManagerInterface */
     private $runAsManager;
+
+    /** @var LoggerInterface */
     private $logger;
 
-    public function __construct(SecurityContextInterface $securityContext, AuthenticationManagerInterface $authenticationManager, AccessDecisionManagerInterface $accessDecisionManager,
-                                AfterInvocationManagerInterface $afterInvocationManager, RunAsManagerInterface $runAsManager, MetadataFactoryInterface $metadataFactory, LoggerInterface $logger = null)
-    {
+    public function __construct(
+        TokenStorageInterface $tokenStorage,
```

Comment:
That's a BC break, you can use something like [this](https://github.com/nelmio/NelmioApiDocBundle/blob/master/Parser/ValidationParser.php#L53) instead.